docs/content/getting-started/quickstart.mdx

```diff
@@ -0,0 +1,198 @@
+---
+title: Dagster Quickstart | Dagster Docs
+description: Run dagster for the first time
+---
+
+# Getting Started with Dagster
+
+<Note>
+  Looking to scaffold a new project? Check out the{" "}
+  <Link href="/getting-started/create-new-project">Creating a New Project</Link>{" "}
+  guide!
+</Note>
+
+## Introduction
+
+Welcome to Dagster! This guide will help you quickly run the [Dagster Quickstart](https://github.com/dagster-io/dagster-quickstart) project, showcasing Dagster's capabilities and serving as a foundation for exploring its features. The [Dagster Quickstart](https://github.com/dagster-io/dagster-quickstart) project can be used without installing anything on your machine by using the pre-configured [GitHub Codespace](https://github.com/features/codespaces)! But if you prefer to run things on your own machine, we've got you covered.
+
+<TabGroup>
+<TabItem name="Option 1: Running Locally">
+
+### Option 1: Running Locally
+
+<DagsterVersion />
+
+Ensure you have one of the supported Python versions installed before proceeding.
+
+Refer to Python's official <a href="https://www.python.org/about/gettingstarted/">getting started guide</a>, or our recommendation of using <a href="https://github.com/pyenv/pyenv?tab=readme-ov-file#installation">pyenv</a> for installing Python.
+
+1. Clone the Dagster Quickstart repository by executing:
+
+   ```bash
+   git clone https://github.com/dagster-io/dagster-quickstart && cd dagster-quickstart
+   ```
+
+2. Install the necessary dependencies using the following command:
+
+   We use `-e` to install dependencies in ["editable mode"](https://pip.pypa.io/en/latest/topics/local-project-installs/#editable-installs). This allows changes to be automatically applied when we modify code.
+
+   ```bash
+   pip install -e ".[dev]"
+   ```
+
+3. Run the project!
+
+   ```bash
+   dagster dev
+   ```
+
+4. Navigate to <a href="localhost:3000">localhost:3000</a> in your web browser.
+
+5. **Success!**
+
+</TabItem>
+<TabItem name="Option 2: Using GitHub Codespaces">
+
+### Option 2: Using GitHub Codespaces
+
+1. Fork the [Dagster Quickstart](https://github.com/dagster-io/dagster-quickstart) repository
+
+2. Select **Create codespace on main** from the **Code** dropdown menu.
+
+<Image
+width={400}
+height={400}
+alt="Create codespace"
+src="/images/getting-started/quickstart/github-codespace-create.png"
+/>
```

Comment:
[2] This image isn't nested under the list item about it (`2`). Indent everything (I think it's 3 spaces; if not that, then 4) and everything should be in line